src/liquid/components/ld-select/ld-select.tsx

```diff
@@ -47,6 +47,9 @@ export class LdSelect implements InnerFocusable {
    */
   @Prop() autofocus = false
 
+  /** Element to attach the popper element to. */
+  @Prop() bodyElement?: HTMLElement | null
```

Comment:
I think I would call this "containerElement" or "portalElement" or something, because there is actually only one body element.